### ISSUE TITLE
Record Attachments Viewer not loading all attachments when pressed

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#4445]
**Describe the bug**
When you click on the attachment viewer icon (top right of form, to the left of the Specify Network icons), only a subset of the total number of attachments are shown.

While every record in a record set might have an attachment, only a small number of the total are shown when clicking on the button. I could

**To Reproduce**
1. Go to this record set (https://sp7demofish.specifycloud.org/specify/view/collectionobject/16/?recordsetid=42)
2. Click on the 'Attachments' icon
    See that it shows **61 attachments**
4. Skip to the end of the record set (record 903 in the set)
5. Click on the 'Attachments' icon
    See that it shows **62 attachments**
6. Go to record 415 
7. Click on the 'Attachments' icon
    See that it shows **124 attachments**

**Expected behavior**
_All_ attachments should be loaded in immediately, as that is the purpose of this button and viewer.

**Screen Recording**

https://github.com/specify/specify7/assets/37256050/6f253b56-00ad-41cf-8fa6-63c7cd0137aa

**Desktop:**
macOS, Arc

**Comment by grantfitzsimmons** [comment]
> Hi. I just noticed the little attachment icon at the top of the Collection Object form, next to the Specify network icons. It seems to open the attachments of the record set in a new navigable screen. If that is what it does, cool! This is great!!
> 
> However, I tried this functionality with a record set that has over 2000 attachments and it is showing only 40:
> 
> ![image](https://github.com/specify/specify7/assets/37256050/c840e942-42ab-49cb-b3f5-9bd7d33d19e1)
> 
> Why may that be?
> 
> Thanks

**Reported By:** [@sorosoro on Discourse](https://discourse.specifysoftware.org/t/attachments-icon-record-set/1505?u=specify)

**Comment by grantfitzsimmons** [comment]
@maxpatiiuk 

You can't load more attachments. If this is the intended behavior, it should not be a button at all. If I could scroll and load more attachments from that record set, I would be satisfied, but it appears that there are only 40 attachments for **all** the records in that set.

Attachments aren't being fetched after the 40 load, and it isn't even trying.

**Comment by maxpatiiuk** [comment]
seems like a bug. I remember @CarolineDenis was adding a button for fetching more attachments

**Comment by CarolineDenis** [comment]
yes that's a bug it was working when releasing this feature. It should load more on scroll like in the attachment gallery tab. The button to fetch more appears in the case there isn't any attachments in the first fetched pull.

**Comment by emenslin** [comment]
Can recreate in edge (7.9.6)
